README.md

```diff
@@ -189,13 +189,25 @@ Authors should respect the `prefers-reduced-motion` media query by conditionally
+
+From the developer's perspective, a portal context can be thought of as having a trivial [session history](https://html.spec.whatwg.org/multipage/history.html#the-session-history-of-browsing-contexts) where only one entry, the current entry, exists. All navigations within the portal are effectively done with replacement. While APIs that operate on session history, such as [window.history](https://html.spec.whatwg.org/multipage/history.html#the-history-interface), can be called within portal contexts, they only operate on the portal's trivial session history. Consequently, portals cannot navigate their hosts using these APIs, unlike iframes.
+
+We want the history model for portals to conform to users' expectations of the back button. Specifically, the back button should take them back to the last thing they saw. To accommodate this, the portal context should not have an independent session history which we aggregate with its host context to present to the user. To illustrate how this could cause problems, consider `example.com/a.html` loading `b.html` in a portal, then navigating the portal to `c.html`, then activating the portal. If the user pressed the back button, it would not be appropriate to navigate the top level page to `b.html`, since this is a state the user has not seen before. Instead, the user is returned to `a.html` (potentially restored from adoption/bfcache) which is the last page they saw. Even for states that the user has seen previously, having a single navigation reintroduce potentially multiple history states maps poorly to the web's chronological history model, as opposed to the mental models for tab switching or mobile device style multitasking.
+
+Navigations within portals are subject to certain security restrictions for loading embedded content ([see above](#permissions-and-policies)). Furthermore, both the portal host and the portal content must use an HTTP(S) scheme, in order to establish their origins.
+
+Navigation errors within portals may cause portal activation to be rejected. Instead of, for example, the user agent showing an error page to the user as with a conventional navigation, the promise returned by the activate method allows a page to gracefully handle the rejection. Furthermore, user agents have existing limitations on navigations initiated by the page where they may be ignored if they are considered to conflict with a user's intent to perform a different navigation. Such cases are not described by the existing navigation spec (see [#218](https://github.com/WICG/portals/issues/218)), but portal activations are subject to these limitations. In the case where another navigation takes precedence over portal activation, the promise returned by the activate method rejects.
+
 TODO:
 
-- One-sentence reiteration of how activation is a navigation.
-- Talk about how session history is tricky.
 - Talk about how bfcache is tricky.
+- Describe reversing transitions on back with bfcache and adopted portals.
```

Comment:
It's looking less and less likely that readoption (re-adoption, not read-option) will be part of an MVP. If so this might be punted for awhile.

Reply:
fwiw I don't think adoption should be required for reverse transitions